src/components/NavBar/DropdownMenu/index.tsx

```diff
@@ -0,0 +1,65 @@
+import NavItem from '../NavItem';
+import { CATEGORIES, SERVICES } from './constants';
+import styles from './styles.module.scss';
+
+function DropsdownMenu() {
+  const handleLink = () => {
+    console.log('links');
+  };
+  const handleLogin = () => {
+    console.log('login');
+  };
+  const handleCategory = () => {
+    console.log('category');
+  };
+  const handleAbout = () => {
+    console.log('about');
+  };
+  const handleContact = () => {
+    console.log('contact');
+  };
```

Comment:
Switch case could be helpful here

Reply:
not yet, just harcoded for test purpose, gonna change in the future.